pkgs/development/tools/misc/lsof/default.nix

```diff
@@ -22,12 +25,14 @@ stdenv.mkDerivation rec {
   };
 
   unpackPhase = "tar xvjf $src; cd lsof_*; tar xvf lsof_*.tar; sourceRoot=$( echo lsof_*/); ";
+  
+  patches = [ ./dfile.patch ];
 
-  preBuild = "sed -i Makefile -e 's/^CFGF=/&	-DHASIPv6=1/;';";
+  preBuild = "sed -i Makefile -e 's/^CFGF=/&	-DHASIPv6=1/;' -e s/-lcurses/-lncurses/ ;";
 
   configurePhase = ''
     # Stop build scripts from searching global include paths
-    export LSOF_INCLUDE=/$(md5sum <(echo $name) | awk '{print $1}')
+    # export LSOF_INCLUDE=/$(md5sum <(echo $name) | awk '{print $1}')
```

Comment:
@adnelson Yes, OK. I just tested `export LSOF_INCLUDE=${stdenv.libc}/include`. It worked well on OS X, and I believe it should in other platforms as well.